src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckDistributionUIConfig.java

```diff
@@ -37,12 +37,15 @@
 import com.synopsys.integration.alert.common.descriptor.config.field.SelectConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.TextInputConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.ui.ProviderDistributionUIConfig;
-import com.synopsys.integration.alert.common.rest.model.CommonDistributionConfiguration;
 import com.synopsys.integration.alert.common.rest.model.FieldModel;
 import com.synopsys.integration.alert.common.rest.model.FieldValueModel;
 
 @Component
 public class BlackDuckDistributionUIConfig extends ProviderDistributionUIConfig {
+    public static final String KEY_FILTER_BY_PROJECT = "channel.common.filter.by.project";
```

Comment:
These don't belong here.  Perhaps ConfigurationJobModel is a better place for these common keys.

Reply:
I agree with Paulo, these dont make sense here because these keys should be common to most or all Provders